### PR TITLE
Add further guidance around ttl

### LINKED ISSUE #107: Add further implementation guidance around when and how to use ttl vs exp claim
Also consider the application and usage of a mechanism like etag to allow a consuming party to determine whether their copy of a given status list is the latest.

### PR DESCRIPTION
**Pull request by tplooker**
Resolves #107

### CODE REVIEW

File: draft-ietf-oauth-status-list.md [hunk 1]
```diff
@@ -815,12 +815,35 @@ A Status List Token in the JWT format should follow the security considerations
...
 
-## Cached and Stale Status Lists
+## Caching and Updates to Status Lists
 
-When Relying Parties fetch the Status List, they need to be aware of its up-to-date status. The 'ttl' (time-to-live) claim
-in the Status List Token provides one mechanism for setting a maximum cache time for the fetched data. This property permits distribution of
-a Status List to a CDN or other distribution mechanism while giving guidance to consumers of the Status List on how often they need to fetch
-a fresh copy of the Status List even if that Status List is not expired.
+When Relying Parties fetch the Status List, they need to be aware of how long it is valid for or in other words how long until they need to check for
+updates to a status list. The 'ttl' (time-to-live) claim in the Status List Token provides one mechanism for setting a maximum cache time for the
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:821]:
```suggestion
updates to a Status List. The 'ttl' (time-to-live) claim in the Status List Token provides one mechanism for setting a maximum cache time for the
```

**Comment by tplooker** [comment]
Note from editors call, perhaps add a little bit of rationale on why we have the ttl and exp claims defined.

File: draft-ietf-oauth-status-list.md [hunk 2]
```diff
@@ -830,12 +832,33 @@ A Status List Token in the JWT format should follow the security considerations
 
 A Status List Token in the CWT format should follow the security considerations of {{RFC8392}}.
 
-## Cached and Stale Status Lists
+## Status List Caching
+
+When fetching a Status List, Relying Parties must carefully evaluate how long a status list is cached for. Collectively the iat, exp and ttl claims when present in a Status List communicate how long a Status List should be cached for. The following diagram illustrates the relationship between these claims and how they are designed to influence caching.
```
Comment (c2bo) [inline draft-ietf-oauth-status-list.md:837]:
```suggestion
When fetching a Status List, Relying Parties must carefully evaluate how long a status list is cached for. Collectively the iat, exp and ttl claims when present in a Status List communicate how long a Status List should be cached and should be considered valid for. The following diagram illustrates the relationship between these claims and how they are designed to influence caching.
```

**Review by c2bo — APPROVED** [review]

File: draft-ietf-oauth-status-list.md [hunk 3]
```diff
@@ -677,6 +677,8 @@ In the case of "application/statuslist+cwt", the response MUST be of type CWT an
 
 The HTTP response SHOULD use gzip Content-Encoding as defined in {{RFC9110}}.
 
+If caching-related HTTP headers are present in the response, Relying Parties SHOULD prioritize the exp and ttl claims within the Status List over the HTTP headers for determining caching behavior.
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:680]:
```suggestion
If caching-related HTTP headers are present in the HTTP response, Relying Parties SHOULD prioritize the exp and ttl claims within the Status List Token over the HTTP headers for determining caching behavior.
```

File: draft-ietf-oauth-status-list.md [hunk 4]
```diff
@@ -830,12 +832,33 @@ A Status List Token in the JWT format should follow the security considerations
 
 A Status List Token in the CWT format should follow the security considerations of {{RFC8392}}.
 
-## Cached and Stale Status Lists
+## Status List Caching
+
+When fetching a Status List, Relying Parties must carefully evaluate how long a status list is cached for. Collectively the iat, exp and ttl claims when present in a Status List communicate how long a Status List should be cached and should be considered valid for. The following diagram illustrates the relationship between these claims and how they are designed to influence caching.
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:837]:
```suggestion
When fetching a Status List Token, Relying Parties must carefully evaluate how long a Status List is cached for. Collectively the `iat`, `exp` and `ttl` claims when present in a Status List Token communicate how long a Status List should be cached and should be considered valid for. The following diagram illustrates the relationship between these claims and how they are designed to influence caching.
```

File: draft-ietf-oauth-status-list.md [hunk 5]
```diff
@@ -830,12 +832,33 @@ A Status List Token in the JWT format should follow the security considerations
...
+  │      │      ttl       │      ttl       │      ttl       │     │
+  │      │ ─────────────► │ ─────────────► │ ─────────────► │ ──► │
+  │      │                │                │                │     │
+  │      │                │                │                │     │
+  │                                                               │
+──┼───────────────────────────────────────────────────────────────┼─►
+  │                                                               │
+~~~
+
+It is essential to understand the distinct purposes of the ttl and exp claims. The ttl claim represents a duration to be interpreted relative to the time the Status List is fetched, indicating when a new version of the Status List may be available. In contrast, the exp claim specifies an absolute timestamp, marking the exact moment when the Status List expires and can no longer be relied upon. Together, these claims provide guidance on when to check for updates (ttl claim) and when the Status List must be refreshed or replaced (exp claim).
```
Comment (paulbastian) [inline draft-ietf-oauth-status-list.md:861]:
```suggestion
It is essential to understand the distinct purposes of the `ttl` and `exp` claims. The `ttl` claim represents a duration to be interpreted relative to the time the Status List is fetched, indicating when a new version of the Status List may be available. In contrast, the `exp` claim specifies an absolute timestamp, marking the point in time when the Status List expires and MUST NOT be relied upon any longer. Together, these claims provide guidance on when to check for updates (`ttl` claim) and when the Status List must be refreshed or replaced (`exp` claim).
```

**Review by paulbastian — APPROVED** [review]

**Pull request merged**